replication-api/src/main/java/org/codice/ditto/replication/api/data/ReplicationStatus.java

```diff
@@ -30,11 +31,13 @@
   String getId();
 
   /**
-   * Get the name of the associated replication configuration
+   * Get the ID of the associated replication configuration
```

Comment:
✏️ Maybe clarify that it gets the unique identifier for the configuration object representing the replicator. Something like that.

Should be `Gets` as opposed to `Get`